chroot/run.go

```diff
@@ -1285,12 +1285,11 @@ func setupChrootBindMounts(spec *specs.Spec, bundlePath string) (undoBinds func(
 			}
 			isReadOnly := statfs.Flags&unix.MS_RDONLY != 0
 			// Check if any of the IDs we're mapping could read it.
-			isAccessible := true
+			isAccessible := false
```

Comment:
can we move this down to have it above the next if statement at line 1293?

Reply:
Good catch! Done :heavy_check_mark:  (and soon to be pushed).